src/org/typo3/chefci/v2/stages/Acceptance.groovy

```diff
@@ -0,0 +1,38 @@
+package org.typo3.chefci.v2.stages
+
+import org.typo3.chefci.helpers.JenkinsHelper
+
+class Acceptance extends AbstractStage {
+
+    Acceptance(Object script, String stageName) {
+        super(script, stageName)
+    }
+
+    @Override
+    void execute() {
+        script.stage(stageName) {
+            testkitchen()
+        }
+    }
+
+    private def testkitchen(){
+        script.node {
+
+            def jenkinsHelper = new JenkinsHelper(script)
+            jenkinsHelper.copyGlobalLibraryScript 'cookbook/.kitchen.docker.yml', '.kitchen.docker.yml'
```

Comment:
if `JenkinsHelper` needs no state to run the `copyGlobalLibraryScript`, a static method would be a better choice to make things more simple - if it needs state, your implementation is the way to go